app/helpers/fs/message_helper.rb

```diff
@@ -0,0 +1,22 @@
+module Fs::MessageHelper
+  def self.build_html_visualization(message)
+    return [ActionController::Base.new.render_to_string('fs/messages/_submission', layout: false, locals: { message: message }), build_html_visualization_from_form(message)].compact.join('<hr>') if message.outbox?
+
+    # TODO: Vieme aj lepsie identifikovat? Nejake dalsie typy v tejto kategorii neexistuju?
```

Comment:
Toto tu je otvorena vec. FS API v response vracia aj `message_type`, da sa identifikovat aj podla toho (ale tiez to je trochu len take hadanie, ked nemame nejaku dokumentaciu k tomu). Nevieme ako casto sa mozu zmenit bud tieto nadpisy alebo `message_type`.
Samotna dorucenka si myslim, ze problem nie je (tam by som sa spolahla aj na `message_type`), ale tieto negativne verzie su narocnejsie. Nevieme ani kolko typov existuje.